integration-tests/olminstall/suite/pipelinerun_naming.py

```
@@ -266,12 +283,12 @@ def build_diagnostic_artifact_log_name(
     """
     Return ``{product}-{version?}-{cluster?}-diagnostic-{datetime}.log``.
 
-    Uses cluster-installed product/version (not ``PRODUCT=existing`` intent).
+    Uses cluster-installed product/version (not test-only PRODUCT intent).
     """
     product_seg = _sanitize_segment(installed_product, max_len=8)
     if not product_seg or product_seg == "unknown":
         raw = (pipeline_product or "").strip().lower()
-        if raw and raw != "existing":
+        if product_installs_operator(raw):
```

Ruff (0.16.2)

[error] Undefined name `product_installs_operator`

(F821)